main/management/commands/kafka_consumer.py

```diff
@@ -6,11 +6,32 @@
 
 from django.conf import settings
 from django.core.management.base import BaseCommand
-from druncschema.broadcast_pb2 import BroadcastMessage
+from druncschema.broadcast_pb2 import BroadcastMessage, BroadcastType
 from kafka import KafkaConsumer
 
 from ...models import DruncMessage
 
+BROADCAST_TYPE_SEVERITY = {
+    BroadcastType.ACK: "DEBUG",
+    BroadcastType.RECEIVER_REMOVED: "INFO",
+    BroadcastType.RECEIVER_ADDED: "INFO",
+    BroadcastType.SERVER_READY: "INFO",
+    BroadcastType.SERVER_SHUTDOWN: "INFO",
+    BroadcastType.TEXT_MESSAGE: "INFO",
+    BroadcastType.COMMAND_EXECUTION_START: "INFO",
+    BroadcastType.COMMAND_RECEIVED: "INFO",
+    BroadcastType.COMMAND_EXECUTION_SUCCESS: "INFO",
+    BroadcastType.DRUNC_EXCEPTION_RAISED: "ERROR",
+    BroadcastType.UNHANDLED_EXCEPTION_RAISED: "CRITICAL",
+    BroadcastType.STATUS_UPDATE: "INFO",
+    BroadcastType.SUBPROCESS_STATUS_UPDATE: "INFO",
+    BroadcastType.DEBUG: "DEBUG",
+    BroadcastType.CHILD_COMMAND_EXECUTION_START: "INFO",
+    BroadcastType.CHILD_COMMAND_EXECUTION_SUCCESS: "INFO",
+    BroadcastType.CHILD_COMMAND_EXECUTION_FAILED: "ERROR",
+    BroadcastType.FSM_STATUS_UPDATE: "INFO",
+}
```

Comment:
One thing I noticed with this current mapping is that we get a mix of INFO and DEBUG statuses for `ps` messages. So "attempting to execute ps" is debug but "successfully executed ps" is info. Seems more consistent if they're both debug. Could you find out  BroadcastType value is being received on the success messages?

Reply:
The two BroadcastType values that were returned are 0 and 6. I think I may have a fix with, but having some issues with my setup, so can't test anything.

Reply:
I addressed the comments below since they are straight forward